app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js

```diff
@@ -832,8 +833,8 @@ define([
          * @private
          */
         _Rewind: function (controls) {
-            controls.find('div[option-id], option[option-id]').removeClass('disabled').removeAttr('disabled');
-            controls.find('div[option-empty], option[option-empty]').attr('disabled', true).addClass('disabled');
+            controls.find('div[data-option-id], option[data-option-id]').removeClass('disabled').removeAttr('disabled');
+            controls.find('div[data-option-empty], option[data-option-empty]').attr('disabled', true).addClass('disabled');
```

Comment:
Please remove `div` from both of these.

Reply:
Can you please tell me why I need to remove div from the selector.
actually div is specify the selector.
if i remove div then it will find [data-option-id] in whole DOM. Instead of finding div with [data-option-id] attribute.

Reply:
`div` must not be used in selectors according to Code Demarcation Standard

Reply:
But I am not changing the div selector.
I am only change the attribute selector to data attribute.
Div is already available in previous code.

Reply:
Ok So I need to change this line as below.

`controls.find('[data-option-id]').removeClass('disabled').removeAttr('disabled');`

right.

Reply:
@niravkrish yep, exactly